src/Tools/Md2Html/Md2HtmlSanitizer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Ganss.XSS;
+
+namespace Md2Html
+{
+    /// <summary>
+    /// HtmlSanitizer class that tracks sanitizing changes
+    /// </summary>
+    internal class Md2HtmlSanitizer : HtmlSanitizer
```

Comment:
do you expect other functionality will go here or should this have a more specific name about "changes"?

Reply:
I don't expect any more functionality. But it is maybe clearer to talk about sanitizing updates and change the property name to ContentWasSanitized? I will do something along those lines.